packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.styles.ts

```diff
@@ -15,14 +15,12 @@ export const getHorizontalBarChartStyles = (props: IHorizontalBarChartStyleProps
       className,
     ],
     items: {
-      height: '32px',
-      marginTop: '5px',
-      position: 'relative',
+      marginBottom: showTriangle ? '16px' : '10px',
     },
     chart: {
       width: '100%',
-      height: '8px',
-      marginBottom: '11px',
+      height: barHeight ? barHeight : 12,
```

Comment:
isn't barHeight fixed?

Reply:
I am not sure but there is a prop for barHeight. Should I ignore it?